0/mdav2/geom.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import masses
+
+class Molecule:
+    """
+    defines the Molecule class. Molecule objects are initialized with a .xyz
+    filename as input. 
+    """
+    def __init__(self,filename):
+        self.moleculefile = open(filename,'r')
+        self.moleculelines = self.moleculefile.readlines()
+        
+        #units are supplied on line 2 
+        self.units = str(self.moleculelines[1]).rstrip()
+        #number of atoms is supplied on line 1   
+        self.natom = int((self.moleculelines[0]).rstrip())
+        
+        #create empty lists of atom labels, masses, charge, and geometry matrix         for later population
+        self.labels = [0]*self.natom
+        self.masses = [int(0)]*self.natom
+        self.charges = [float(0)]*self.natom
```

Comment:
`0` is already an int, no need to cast it. If you want to make it a float, use 0.0. However, there is no need to do so here.